### PR TITLE
Adding details at CFP page

### PR DESCRIPTION
**Pull request by railsstudent**
Related to #11
I created a second card to display details in Chinese.

### CODE REVIEW

File: assets/pages/cfp/index.jinja [hunk 1]
```diff
@@ -49,16 +49,75 @@
...
+                      </li>
+                  </ul>      
+                </p>               
+                <p>
+                  If you any any enquiries and suggestions, please feel freely to contact with our program team at conference@opensource.hk.
+                </p>
+            </div>            
+        </div>
+        
+        <div class="card">
```
Comment (tonyhhyip) [inline assets/pages/cfp/index.jinja]: Add class `blue-theme` and `yellow-theme-text` for this card

File: assets/pages/cfp/index.jinja [hunk 2]
```diff
@@ -49,16 +49,75 @@
...
+                <p>感謝你支持香港開源年會 (HKOSCon) ，你的擁躍支持幫助過去四年 HKOSCon 得以成功舉行。</p>
+                <p>我們剛展開第五屆香港開源年會的稿件收集 CFP，希望你和朋友能再次投稿，多少無拘，今年再來參加 HKOSCon。</p>
+                <ul class="browser-default">
+                    <li>會議日期: 2017 年 6 月 9-10 日 (五－六)</li>
+                    <li>場地及贊助: 香港數碼港 Cyberport</li>
+                    <li>暫定形式: 星期五單一主軌，星期六雙主軌，另有三間小房作演講及工作坊等活動</li>
+                    <li>預計參加人數: 350 (與 2016 年出席人數相約)</li>
+                    <li>
+                        Link:
+                        <a href="https://goo.gl/forms/fmNjTshYMZlaaKo02" target="_blank">
```
Comment (tonyhhyip) [inline assets/pages/cfp/index.jinja]: Add class `link`

File: assets/pages/cfp/index.jinja [hunk 3]
```diff
@@ -49,16 +49,75 @@
...
+                </p>
                 <ul class="browser-default">
-                    <li>Deadline: 12 Feb, 2017 (Sat)</li>
+                    <li>Date of Conference: 9-10 June 2017 (Fri - Sat)</li>
+                    <li>Venue & Sponsor: Cyberport, Hong Kong.</li>
+                    <li>Tentative Format: Single main track on Friday, 2 main tracks on Saturday. Additional  workshops, talks and activities will be also available on additional 3 tracks at small rooms.</li>
+                    <li>Targeted number of participants: 350 (same as number of attendance in 2016)</li>
                     <li>
                         Link:
                         <a href="https://goo.gl/forms/fmNjTshYMZlaaKo02" target="_blank">
```
Comment (tonyhhyip) [inline assets/pages/cfp/index.jinja]: Add class link

**Review by tonyhhyip — CHANGES_REQUESTED** [review]
Please add some class for styling

**Comment by railsstudent** [comment]
Done.
Add new style.scss class to add custom style

File: assets/pages/cfp/index.jinja [hunk 4]
```diff
@@ -44,11 +44,11 @@
             </div>
         </div>
 
-        <div class="card">
-            <div class="title">
+        <div class="card blue-theme">
```
Comment (tonyhhyip) [inline assets/pages/cfp/index.jinja]: Please refers to https://github.com/404busters/hkoscon-2017/blob/master/assets/pages/about/coc.jinja

File: assets/pages/cfp/index.jinja [hunk 5]
```diff
@@ -88,11 +88,11 @@
             </div>            
         </div>
         
-        <div class="card">
-            <div class="title">
+        <div class="card blue-theme">
```
Comment (tonyhhyip) [inline assets/pages/cfp/index.jinja]: Please refers to https://github.com/404busters/hkoscon-2017/blob/master/assets/pages/about/coc.jinja

File: assets/scss/style.scss [hunk 6]
```diff
@@ -0,0 +1,13 @@
+
+.card {
```
Comment (tonyhhyip) [inline assets/scss/style.scss]: Please modified in https://github.com/404busters/hkoscon-2017/blob/master/assets/scss/elements/_card.scss

**Review by tonyhhyip — CHANGES_REQUESTED** [review]
Not expected result

**Review by tonyhhyip — APPROVED** [review]
Alright, I will handle the rest

**Pull request merged**